package.json

```diff
@@ -10,7 +10,8 @@
     "build": "npm run build:css && npm run build:js",
     "build:js": "browserify site/js/app.js -o site/assets/bundle.js -t [ babelify --presets [es2015] ]",
     "build:css": "postcss --use stylelint --use postcss-import --postcss-import.path 'node_modules/prismjs/themes' --use postcss-cssnext --use postcss-reporter -o site/assets/styles.css site/css/*.css",
-    "watch": "postcss --watch --use stylelint --use postcss-import --postcss-import.path 'node_modules/prismjs/themes' --use postcss-cssnext --use postcss-reporter -o site/assets/styles.css site/css/*.css"
+    "watch": "postcss --watch --use stylelint --use postcss-import --postcss-import.path 'node_modules/prismjs/themes' --use postcss-cssnext --use postcss-reporter -o site/assets/styles.css site/css/*.css",
+    "start": "node_modules/.bin/http-server ."
```

Comment:
Just a quick little note. NPM scripts actually have access to the `node_modules/.bin` directory so you only have to use `http-server` instead of the full path! 👍